docs/install/knative-with-operators.md

```diff
@@ -1,84 +1,74 @@
 ---
-title: "Installing Knative components using Operator"
-weight: 10
+title: "Knative Operator installation"
+weight: 02
 type: "docs"
+showlandingtoc: "false"
 ---
 
-Knative provides an [operator](https://github.com/knative/operator) as a tool to install, configure and manage Knative. The Knative operator leverages custom objects
-in the cluster to define and manage the installed Knative software. This guide explains how to install and uninstall
-Knative using Knative operator.
+Knative provides a [Kubernetes Operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) to install, configure and manage Knative.
 
-## Before you begin
-
-Knative installation using the Operator requires the following:
-
-- A Kubernetes cluster v1.16 or newer, as well as a compatible kubectl. This guide assumes that you've already created
-a Kubernetes cluster. If you have only one node for your cluster, set CPUs to at least 6, Memory to at least 6.0 GB,
-Disk storage to at least 30 GB. If you have multiple nodes for your cluster, set CPUs to at least 2, Memory to at least
-4.0 GB, Disk storage to at least 20 GB for each node.
-- The Kubernetes cluster must be able to access the internet, since the Knative operator downloads images online.
-- [Download and install Istio](./installing-istio.md).
-
-## Limitations of Knative Operator:
-
-Knative Operator is still in Alpha phase. It has not been tested in a production environment, and should be used
+**NOTE:** The Knative Operator is still in Alpha phase. It has not been tested in a production environment, and should be used
 for development or test purposes only.
 
-## Install Knative with the Knative Operator
+## Prerequisites
 
-You can find the release information of Knative Operator on the [Releases page](https://github.com/knative/operator/releases).
+- You have a cluster that uses Kubernetes v1.17 or newer.
```

Comment:
This updated v1.16 to v1.17, but the minimum version is actually now v1.18 I think